collection-scripts/gather_services_status

```diff
@@ -68,6 +71,19 @@ get_openstack_status() {
     run_bg ${BASH_ALIASES[os]} network agent list '>' "$BASE_COLLECTION_PATH"/ctlplane/network_agent_list
 }
 
+# Rabbitmq info gathering -
+get_rabbitmq_status() {
+    local RABBIT_PATH="$BASE_COLLECTION_PATH/ctlplane/rabbitmq"
+    mkdir -p "$RABBIT_PATH"
+    rabbit_instances=$(oc get pods -l "$RABBITMQ_SELECTOR" -o custom-columns=NAME:.metadata.name --no-headers)
+    for instance in $rabbit_instances; do
+        mkdir -p "$RABBIT_PATH/$instance"
+        run_gb oc -n "${OSP_NS}" rsh "$instance" rabbitmqctl status '>' "$RABBIT_PATH"/"$instance"/status
+        run_gb oc -n "${OSP_NS}" rsh "$instance" rabbitmqctl list_queues '>' "$RABBIT_PATH"/"$instance"/list_queues
+        run_gb oc -n "${OSP_NS}" rsh "$instance" rabbitmqctl list_connections '>' "$RABBIT_PATH"/"$instance"/list_connections
```

Comment:
@lmiccini is the above enough for rabbit?

Reply:
I would add:
rabbitmqctl list_policies
rabbitmqctl cluster_status

and potentially:
rabbitmqctl list_unresponsive_queues

Reply:
Done